testingenv/centos8/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7
```

Comment:
Found while checking the logs

```suggestion
FROM centos:8
```

It doesn't however solve our issue with dependencies.

Reply:
I guess `centos:8` might fix that strange ruby dependency in tests mentioned below.